### PR TITLE
Review and update the readme

### PR DESCRIPTION
**Pull request by ijdickinson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,17 +198,107 @@ Note that, by default, SPARQL queries will be run against the dev triple store.
...
+- a `Makefile` is the key interface between the deployment automation and the
+  application code, especially `make image` and `make tag`.
+
+The determination of which Docker images go to which environments is managed by
+the `deployment.yml` file. At the time of writing:
+
+- tags matching `vNNN`, e.g. `v1.2.3`<br />
+  Images built from this tag will be deployed to production environments
+- branch `dev-infrastructure`<br />
+  Images built from this branc will be deployed to the `dev` environment
```
Comment (der) [inline README.md:223]: branch

In two minds about having this in the README and so having to maintain in two places and risk of getting out of sync. This config will need to change a lot in the next few weeks. OTOH once it's stable it's more accessible in the README. On balance OK with leaving it here.

File: README.md [hunk 2]
```diff
@@ -135,17 +198,107 @@ Note that, by default, SPARQL queries will be run against the dev triple store.
...
+  Folder where the subscribers to the known ActiveSupport notifications are
+  defined. This is where the transform from `ActiveSupport::Notification` to
+  Prometheus counter or gauge is performed.
+
+In addition to the metrics we define, there is a collection of standard
+metrics provided automatically by the
+[Ruby Prometheus client](https://github.com/prometheus/client_ruby)
+
+To test Prometheus when developing locally, there needs to be a Prometheus
+server running. Tip for Linux users: do not install the Prometheus apt
```
Comment (der) [inline README.md:280]: Aside: probably says something about me that when I'm checking I've set up a new prometheus endpoint OK I use `curl` :)

**Review by der — APPROVED** [review]
:+1:

File: README.md (hunk 2 above)
Comment (ijdickinson) [reply README.md:280]: To be honest, so do I (well, more often I'll use the browser). I wrote this when I was first getting going with Prometheus, as I wanted to understand the usage of the metrics to give some context to the design decisions.

File: README.md (hunk 1 above)
Comment (ijdickinson) [reply README.md:223]: Agree with the maintenance question, but I think a key audience for this document is new devs, or people who don't work on this codebase very often. It's trivial to look in `deployment.yml` for the deployment info, once you know or remember that `deployment.yml` is a thing :smile:

**Review by jonrandahl — APPROVED** [review]
Outside of @der's comments looks good to me! 👍

File: README.md (hunk 1 above)
Comment (ijdickinson) [reply README.md:223]: > branch

We're now in the Wordle-cene era. Words are adapting to become 5 letters long :grin:

**Pull request merged**